Ginger/GingerCore/SourceControl/SVNSourceControl.cs

```diff
@@ -265,18 +265,20 @@ public override bool GetLatest(string path, ref string error, ref List<string> c
                 RepositoryFolderBase repositoryFolderBase = null;
                 if (WorkSpace.Instance.SolutionRepository != null)
                 {
-                    repositoryFolderBase = WorkSpace.Instance.SolutionRepository.GetRepositoryFolderByPath(Path.GetDirectoryName(path));
+                    if (Path.GetDirectoryName(path) != SolutionFolder)
+                    {
+                        repositoryFolderBase = WorkSpace.Instance.SolutionRepository.GetRepositoryFolderByPath(Path.GetDirectoryName(path));
+                    }
+                    else
+                    {
+                        repositoryFolderBase = WorkSpace.Instance.SolutionRepository.GetRepositoryFolderByPath(path);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this useless assignment to local variable 'repositoryFolderBase'.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5101775)